dashboard/modules/job/tests/test_job_agent.py

```diff
@@ -518,8 +518,14 @@ def test_agent_logs_not_streamed_to_drivers():
     err_str = proc.stderr.read().decode("ascii")
 
     print(out_str, err_str)
-    assert "(raylet)" not in out_str
-    assert "(raylet)" not in err_str
+
+    def check_output(blob):
```

Comment:
Not quite sure, but the test is to make sure no logs are streamed to worker from raylet. SHould we keep that?

Reply:
But we added new logs in this change that aren't errors, so we broke that invariant. Probably doesn't make sense to test for it anymore

Reply:
ok fixed